projects/widgets/docs/linken-van-artikels-aan-events-via-widgets.md

```diff
@@ -0,0 +1,123 @@
+
+### 2. Het unieke ID van een evenement vinden
+
+De waarde van `data-cdbid` uit het `<script>`tag bepaalt welk evenement ingeladen wordt. In het voorbeeld van hierboven is dit het evenement met cdbid `ffe4f207-2f8d-4dce-87a6-08abf59aefbe`.
+
+Een redacteur hoeft dus enkel te kiezen welk evenement hij wilt integreren in zijn artikel, door de waarde voor `data-cdbid` aan te passen. De waarde voor `data-cdbid` kan je terugvinden op verschillende manieren.
+
+* **in de agenda van UiTinVlaanderen**
+
+Je kan naar de detailpagina van het evenement op UiTinVlaanderen surfen. In de url vind je het `cdbid` van het evenement:
+
+![redactionele-tools-detail-uiv.png](../assets/images/redactionele-tools-detail-uiv.png)
+
+Dit `cdbid` dien je vervolgens te kopiëren en te gebruiken als waarde voor `data-cdbid` in het widget `<script>` tag.
+
+* **in de eigen UiT-agenda**
+
+Je kan naar de detailpagina van het evenement in de eigen UiT-agenda surfen. In de url vind je het `cdbid` van het evenement:
+
+![redactionele-tools-detail-bruzzwebsite.png](../assets/images/redactionele-tools-detail-bruzzwebsite.png)
+
+Dit `cdbid` dien je vervolgens te kopiëren en te gebruiken als waarde voor `data-cdbid` in het widget `<script>` tag.
+
+* **door de UiTdatabank te doorzoeken vanuit je eigen CMS**
+
+Je kan ook een zoek bouwen doorheen evenementen vanuit je eigen CMS.
+
+`[VIDEO]`
+
+## Vereisten
+
+### Inhoudelijke vereisten
+
+De artikels die je linkt moeten voldoen aan een aantal inhoudelijke vereisten.
+
+#### Wordt toegelaten
+
+* ✅ Een reportage of interview met de maker(s) of artiesten van een voorstelling of activiteit, naar aanleiding van een evenement dat nog moet plaatsvinden
+* ✅ Artikel waarin één of meerdere komende activiteiten getipt worden
+* ✅ Recensie van een activiteit waarvoor nog voorstellingen of toonmomenten in de toekomst plaatsvinden
+
+#### Wordt niet toegelaten
+
+* ❌ Zuiver promotionele redactionele content voor evenementen die als organisator zelf organiseert
+* ❌ redactionele content die geen meerwaarde toevoegt bovenop de aankondiging van het evenement zelf
+* ❌ redactionele content waarbij teruggeblikt wordt naar een afgelopen evenement en waarvoor geen link is met een evenement in de toekomst (bijvoorbeeld de recensie van activiteit die niet meer terugkeert)
+
+### Technische vereisten
+
+Om artikels te kunnen linken via widgets moet je website voldoen aan volgende technische vereisten.
+
+#### Elk artikel moet een aantal vaste HTML tags bevatten
+
+* Een `<h>` tag met daarin de titel van het artikel
+* Een `<p>` tag bevatten met daarin de inleiding van het artikel
+* Een `<meta>` tag bevatten met daarin `description`als waarde voor het attribuut `name`:
+
+```html
+<meta name="description" content="Slongs: sociaal bewogen hangen">
+```
+
+#### Aanwezigheid van een favicon
+
+Dit favicon zal gebruikt worden om je artikels extra onder de aandacht te brengen in de agenda's waar je artikels gepubliceerd worden.
```

Comment:
Do we have an example of this? And are there guidelines for the favicon? Traditionally these were only 16x16 pixels, but these days they can have different dimensions I think (while the 16x16 version is technically also still a valid favicon). 

Maybe @brampauwelyn or @simon-debruijn can also chime in on that with more info.

Reply:
There are no requirements for the favicon. The scraping process just checks the favicon, regardless the resolution.